interceptor.go

```diff
@@ -235,17 +235,45 @@ func (i *interceptor) WrapStreamingClient(next connect.StreamingClientFunc) conn
 			protocol: protocolToSemConv(conn.Peer()),
 			attrs:    attributesFromRequest(req),
 		}
+		carrier := propagation.HeaderCarrier(conn.RequestHeader())
+		ctx = i.config.propagator.Extract(ctx, carrier)
+		spanCtx := trace.SpanContextFromContext(ctx)
+		tracer := i.config.Tracer()
+		name := strings.TrimLeft(conn.Spec().Procedure, "/")
+		ctx, span := tracer.Start(
+			trace.ContextWithRemoteSpanContext(ctx, spanCtx),
```

Comment:
Shouldn't be `ContextWithRemoteSpanContext` because that implies that it would be a remote span and _not_ a local span

Reply:
Resolved in ebe6bd9ccb0038f11b8b48d1dd98ea5abe9e930f